quic/s2n-quic-rustls/Cargo.toml

```diff
@@ -12,7 +12,7 @@ exclude = ["corpus.tar.gz"]
 
 [dependencies]
 bytes = { version = "1", default-features = false }
-rustls = { version = "0.21", features = ["quic"] }
+rustls = "0.23"
 rustls-pemfile = "1"
```

Comment:
Do we also need to upgrade this dependency (https://github.com/aws/s2n-quic/pull/2053)? Or maybe we try to do that separately since it doesn't seem like we have any issues here?

Reply:
I was thinking separately. I saw a some breaking changes and didnt pursue it further to see if its going to be a trivial update since you also mentioned it here.

Reply:
updated in https://github.com/aws/s2n-quic/pull/2143/commits/af3b5342ee67e291aa6a755543cd68ddfe49cbb6